src/components/content-tab-settings.tsx

```diff
@@ -63,13 +65,43 @@ export function ContentTabSettings( { selectedSite }: ContentTabSettingsProps )
 					</SettingsRow>
 					<SettingsRow label={ __( 'Local URL' ) }>
 						<CopyTextButton
-							text={ `http://${ domain }` }
+							text={ `${ protocol }://${ domain }` }
 							label={ `${ domain }, ${ __( 'Copy site url to clipboard' ) }` }
 							copyConfirmation={ __( 'Copied!' ) }
 						>
 							{ domain }
 						</CopyTextButton>
 					</SettingsRow>
+					<SettingsRow label={ __( 'SSL' ) }>
+						<div>
+							<span>{ selectedSite.enableSSL ? __( 'Enabled' ) : __( 'Disabled' ) }</span>{ ' ' }
+							{ selectedSite.enableSSL && (
+								<Button variant="link" onClick={ () => getIpcApi().openCertificate() }>
+									{ __( 'Trust Certificate' ) }
+								</Button>
+							) }
+						</div>
+						{ selectedSite.enableSSL && (
+							<div className="mt-1">
```

Comment:
```suggestion
							<div className="mt-1 max-w-96">
```
Makes for a better and more readable look